lib/install/validate/config.go

```diff
@@ -509,62 +508,80 @@ func (v *Validator) checkLicense(op trace.Operation) error {
 	return nil
 }
 
-func (v *Validator) checkEvalLicense(op trace.Operation, licenses []types.LicenseManagerLicenseInfo) {
```

Comment:
How come we're no longer warning about eval licenses?

Reply:
we can add it back..but there are a myriad of things that could be changed that would impact the working of vic.  Should we just warn that if they make any license changes it could impact...as it's not limited to eval...in addition we were attempting (I thought) to move away from a rigid license check and be more feature based...